services/slack/intermediate_test.go

```diff
@@ -302,7 +314,7 @@ func TestTransformBigGroupChannels(t *testing.T) {
 	for i := range result {
 		assert.Equal(t, fmt.Sprintf("purpose%d", i+1), result[i].Name)
 		assert.Equal(t, fmt.Sprintf("purpose%d", i+1), result[i].DisplayName)
-		assert.Equal(t, channelMembers, result[i].Members)
+		assert.Equal(t, len(channelMembers), 9)
```

Comment:
I don't love this check here, but without reworking the entire test this is the only way i can see.

Reply:
Should be this though right?

```suggestion
		assert.Equal(t, len(result[i].Members), 9)
```